app/controllers/admin/tools_controller.rb

```diff
@@ -40,8 +40,8 @@ def handle_article_cache
     end
 
     def bust_link(link)
-      if link.starts_with?("https://#{ApplicationConfig['APP_DOMAIN']}")
-        link.sub!("https://#{ApplicationConfig['APP_DOMAIN']}",
+      if link.starts_with?(URL.url)
+        link.sub!(URL.url,
```

Comment:
Seems the formatting got a bit funky here, the second argument was moved to the next line.

Reply:
Fixed